cds_ils/literature/views.py

```diff
@@ -0,0 +1,32 @@
+#  -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS Literature views."""
+
+from flask import Blueprint, redirect
+from invenio_app_ils.proxies import current_app_ils
+from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_records_rest.errors import PIDDoesNotExistRESTError
+
+legacy_blueprint = Blueprint("cds_ils_legacy", __name__)
+
+
+@legacy_blueprint.route("/legacy/<id>")
+def legacy_redirect(id):
+    """Redirect to the documents."""
+    try:
+        Document = current_app_ils.document_record_cls
+        record = Document.get_record_by_legacy_recid(id)
```

Comment:
The legacy recid can be a Document or a Series, in my opinion we should bind it to the Document.
I would change this to do:

```
from invenio_pidstore.resolver import Resolver
from invenio_records.api import Record

try:
    resolver = Resolver(
            pid_type=LEGACY_PID_TYPE, object_type="rec", getter=Record.get_record
        )
    _, record = resolver.resolve(str(pid_value))
except PersistentIdentifierError:
    return ...404...

is_document = schema.endswith(Document._schema)
is_series = schema.endswith(Series._schema)
assert is_document or is_series

url_path = "documents" if is_document else "series"

return redirect("/api/{0}/{1}".format(record["pid"], url_path))
```